tox.ini

```diff
@@ -1,20 +1,19 @@
 [tox]
-envlist = py311,lint,system
 skipsdist = true
 [pytest]
 norecursedirs = env
 markers =
-	slow: marks tests as slow (use '-- runslow' to run)
+	slow: marks tests as slow
```

Comment:
I saw you removed a lot of those. Is this still used?

Reply:
there are 3 slow tests in the test suite, those tests takes 1-3 min each

When I code I use:

    pytest -m "not slow"

this takes a ~3 minute. Then i use --lf (last failed)

    pytest  --lf -m "not slow"

You can also combine slow and system:

    pytest -m "not slow and not system"